### ISSUE TITLE
Change CLI Command 'geoips get' to 'geoips describe'

### ISSUE THREAD

**Issue by evrose54** [NRLMMD-GEOIPS/geoips#495]
# Requested Update

### Description
We have decided that ``geoips get`` is not the best nomenclature for what that command actually does. Instead, we as developers have decided that ``geiops get`` should be renamed to ``geoips describe``, as that is more closely linked to what the command actually does. Since there are PR's that depend on one another, we've decided to leave this as a separate, small PR which we can implement after #444, #465, and #478 have been merged.

### Background and Motivation
This issue stems from this [comment](https://github.com/NRLMMD-GEOIPS/geoips/pull/444/files#r1541849766) left on PR #444.

### Checklist for Completion
* [ ] Update all references of ``geoips get`` to ``geoips describe``, including the help instructions and documentation.

**Comment by evrose54** [comment]
I have a branch ready for this but am waiting until all other CLI PRs have been merged. Looks like way too many files have been changed and this will be fixed once those are merged into main.
